src/app/pages/system/general-settings/support/support-card/support-card.component.html

```diff
@@ -0,0 +1,7 @@
+<mat-card class="system-card" gdArea="system-card">
+  <mat-toolbar-row class="title-bar">
+    <h3>{{ supportTitle | translate }}</h3>
+  </mat-toolbar-row>
+
+  <ix-support></ix-support>
```

Comment:
Feels like you can move `ix-support` here. No real value in this wrapper component.